src/app/components/feed/FeedHeader.module.css

```diff
@@ -0,0 +1,39 @@
+.feedHeader {
+  margin: 16px 0;
+
+  .feedHeaderSubtitle {
+    text-transform: uppercase;
+  }
+
+  .feedHeaderTitle {
+    color: var(--textPrimary);
+    margin: 0;
+  }
+
+  .feedHeaderRow {
+    align-items: center;
+    display: flex;
+    gap: 12px;
+  }
+
+  .feedHeaderIcons {
+    align-items: center;
+    display: flex;
+    gap: 12px;
+  }
+
+  .feedHeaderIcon {
```

Comment:
please add a font-size of 20px so the icon matches the designs. once the settings icon has been changed to the local svg version it will size correctly

Reply:
Done here: https://github.com/meedan/check-web/commit/d45bfc0253b418cd0293b1bd1267300dde646709